frontend/src/metabase/admin/datamodel/containers/RevisionHistoryApp.jsx

```diff
@@ -32,7 +32,7 @@ export default connect(mapStateToProps, mapDispatchToProps)(RevisionHistoryApp);
 class SegmentRevisionHistoryInner extends Component {
   render() {
     const { segment, ...props } = this.props;
-    return <RevisionHistory object={segment} {...props} />;
+    return <RevisionHistory segment={segment} {...props} />;
```

Comment:
The actual fix.

Reply:
The actual fix was to remove `Tables.load` which injected `object` prop into `RevisionHistory` overriding whatever was being passed here.

Reply:
The fix was working even before I removed Tables simply by changing the name of the prop from `object` to `segment`. Tables were overriding the `object` prop.